pre_commit/git.py

```diff
@@ -37,9 +37,10 @@ def is_in_merge_conflict():
 def parse_merge_msg_for_conflicts(merge_msg):
     # Conflicted files start with tabs
     return [
```

Comment:
yeah no, I just updated git on my home machine and couldn't get the tests passing without this.